### ISSUE TITLE
endpadding is present in docs and code, but has no doc string

### ISSUE THREAD

**Issue by monfera** [plotly/documentation#730]

**Comment by cldougl** [comment]
a `description` will need to be added to the js repo: https://github.com/plotly/plotly.js/blob/43d3710bdb183f9c1681f40efe8649a7238efce5/src/plots/polar/axis_attributes.js#L73
(the attribute reference is generated from the plot schema in plotly.js)

**Comment by monfera** [comment]
Yes I hesitantly put it here rather than the plotly.js repo as end result is user doc

**Comment by monfera** [comment]
There are quite a few missing ones in pie as well: https://plot.ly/javascript/reference/#pie-marker-line-colorsrc

**Comment by cldougl** [comment]
@monfera yep either repo is fine for this issue imo
